Cmdr/CmdrClient/CmdrInterface/Window.lua

```diff
@@ -243,7 +243,16 @@ function Window:BeginInput(input, gameProcessed)
 		return
 	end
 
-	if input.KeyCode == Enum.KeyCode.Down then -- Auto Complete Down
+	if table.find({Enum.UserInputType.MouseButton1,Enum.UserInputType.MouseButton2,Enum.UserInputType.Touch},input.UserInputType) then
+		local ps = input.Position
+		local xPos = Gui.AbsolutePosition.X
+		local yPos = Gui.AbsolutePosition.Y
+		local xOff = Gui.AbsoluteSize.X
+		local yOff = Gui.AbsoluteSize.Y
+		if not(ps.X >= xPos and ps.X <= xPos+xOff and ps.Y >= yPos and ps.Y <= yPos+yOff) then
+			self:Hide()
+		end
+	elseif input.KeyCode == Enum.KeyCode.Down then -- Auto Complete Down
 		self:SelectVertical(1)
```

Comment:
Please space infix operators correctly as inline with the style of the rest of the project

Reply:
I've spaced out the plus operators as requested. If I didn't do the inline bit correctly I'm afraid I don't quite understand what needs to happen. Still keeping this as a draft, as I'm looking to allow the user to configure this.